Chat/src/app/utils/getUserDetailsFromURL.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export type UserDetails = {
+  userId: string;
+  token: string;
+  displayName: string;
+};
+
+/**
+ *
+ * The user details of an existing participant are extracted from the url
+ * using URLsearchparams.
+ *
+ * @returns The current userId, accessToken and display name as an object
+ *
+ */
+export const getUserDetailsFromURL = (): UserDetails | null => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const userId = urlParams.get('userId');
+  const token = urlParams.get('token');
+  const displayName = urlParams.get('displayName') ?? '';
```

Comment:
If I understand this correctly, the user of the sample would have to input their token, userId, and their display name in as query args in the url to join an existing thread as an existing user. I think that is not a pleasant user experience. 

I would prefer if we had a button in the HomeScreen to say 'Join Chat'. When clicked it would go to another screen where you can input token, userId, and their display name in textboxes (which can be masked)